### DISCUSSION TITLE
Brew wont update

### DISCUSSION THREAD

**Discussion by nikixuy** [Homebrew/discussions#922] [Getting started]
After I install brew, i want to update it. I am gettin this ! Help me please I am new to both mac and using terminal
Nikis-MacBook-Pro ~ % brew install upx 
zsh: command not found: brew
Nikis-MacBook-Pro ~ %

**Comment by carlocab** [comment]
Did you follow the instructions at the end of the install to add `brew` to your `PATH`?

**Reply by nikixuy** [reply]
how can i find it ?
i just copy and paste the brew code from site

**Reply by carlocab** [reply]
Please pay attention to the messages that `brew` shows you when you install something. The important ones are shown at the end.

The installer should've shown you something _like_ the following message:
```
==> Next steps:
- Add Homebrew to your PATH in /Users/homefolder/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
- Run `brew help` to get started
```

The command you want to run is `echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> $HOME/.zprofile`. Restart your terminal for this to take effect.

**Reply by nikixuy** [reply]
Thank you so much for you respond. this happens : 
nikix@Nikis-MacBook-Pro ~ % echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
zsh: no such file or directory: /Users/homefolder/.zprofile
nikix@Nikis-MacBook-Pro ~ % echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> $HOME/.zprofile
nikix@Nikis-MacBook-Pro ~ %

**Reply by nikixuy** [reply]
I am not sure if i am doing it right. should i install brew all over again >?

**Reply by carlocab** [reply]
This is why I said

>The installer should've shown you something _like_ the following message

Emphasis on **like**. You weren't meant to run that command, and I told you after which command you needed to run. `brew` should be working now. If it isn't, either there is something wrong with your shell, or you didn't do this:

>Restart your terminal for this to take effect.

Please don't just input random commands into your terminal.

**Reply by nikixuy** [reply]
how can i be sure that brew is working and it is update ? what code should i type

**Reply by carlocab** [reply]
You can just run `brew help` to see if it outputs the help message. You can also check that `brew update` is working.

**Reply by nikixuy** [reply]
yeeeees ! thanks it is working ! so my app is not working . i installed brew to be able to install Ableton live 11 patch . I am getting "you don't have permission to open app" error

**Reply by nikixuy** [reply]
and again thank you for your time again
